ansible/roles/test/tasks/acltb.yml

```diff
@@ -76,18 +76,54 @@
         errors_expected: false
       include: roles/test/tasks/run_command_with_log_analyzer.yml
 
+    - name: Get the total ACL table size before populating the ACL table
+      command: "{{item}}"
+      with_items:
+        - crm config polling interval 5
+        - crm show resources acl table
+      register: out
+
+    - name: Extract out the total available acl entry count from CRM show
+      set_fact:
+        count: "{{out.results[1].stdout_lines[4].split()}}"
+
+    - name: Convert the total count value to integer
+      set_fact:
+        total: "{{count[3] | int}}"
+
+    - name: Subtracting configured rules from total rules
+      set_fact:
+        expected: "{{total|int - 14}}"
```

Comment:
Is the value "14" you got from the file acltb_test_rules_part_1.json and part2.json? If so, it is slightly tricky that we've to modify here if there are any new add/delete in those files. Just a thought!

Reply:
ansible has a json query filter which i think we can use to find the ACL rule count in part2.json. but this filter is available only from 2.2. We are using 2.0.0.2

http://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#json-query-filter